plugins/flytekit-greatexpectations/setup.py

```diff
@@ -4,7 +4,7 @@
 
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
-plugin_requires = ["flytekit>=0.21.0,<1.0.0", "great-expectations>=0.13.23"]
+plugin_requires = ["flytekit>=0.21.0,<1.0.0", "great-expectations==0.13.28"]
```

Comment:
@samhita-alla can we actually change it atleast to be a range, with  upperbound of 0.13.28. Better would be to actually fix it for the new version, since we have not really announced anything

Reply:
I believe https://github.com/great-expectations/great_expectations/issues/3290 is the issue.

Reply:
We can proceed with 0.13.28 for now. I'll bump the version later.